database.txt

```diff
@@ -1,38 +0,0 @@
- +-------------+      +-----------------+   +---------------+   +------------+
- | users       |      | preferences     |   | filters       |   | rules      |
```

Comment:
Was it made by hand?

I was thinking it would be easiest to build the database diagram from the database itself or the SQLAlchemy models. I found [this](https://bitbucket.org/zzzeek/sqlalchemy/wiki/UsageRecipes/SchemaDisplay), which produces:
![dbschema](https://cloud.githubusercontent.com/assets/1977525/20928810/6b3e087e-bb95-11e6-9c33-be5947bbfde5.png)

It's not great, admittedly, but it's automated and will always be up-to-date. Maybe there are better tools out there, as well, but that one was super simple to use and would be easy to add to the docs Makefile.